src/dsync/dsync.c

```diff
@@ -1197,14 +1206,20 @@ static int dsync_strmap_compare_lite(
         uint64_t src_mtime_nsec = mfu_flist_file_get_mtime_nsec(src_compare_list, idx);
         uint64_t dst_mtime      = mfu_flist_file_get_mtime(dst_compare_list, idx);
         uint64_t dst_mtime_nsec = mfu_flist_file_get_mtime_nsec(dst_compare_list, idx);
+        char srcmtimeString[100];
+        char dstmtimeString[100];
+        strftime(srcmtimeString, sizeof(srcmtimeString), "%c", localtime(&src_mtime));
```

Comment:
I think these two strftime() calls might affect performance since they are called unconditionally.  How about declaring and filling these string values only if you need them?

Reply:
debug statement left out, taking it out. thanks for the findings.